custom_components/terncy/hass/add_entities.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import TYPE_CHECKING
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .entity import TerncyEntity
+from .entity_descriptions import TerncyEntityDescription
+from ..const import DOMAIN
+
+if TYPE_CHECKING:
+    from ..core.gateway import TerncyGateway
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def create_entity(
+    gateway: "TerncyGateway",
```

Comment:
define a const for it? as it's used in multi places.

Reply:
It's not const, the quote is only for type checking, avoid circular references.